content/millennium/r4/scheduling/slot.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Slot | R4 API
+---
+
+# Slot
+
+* TOC
+{:toc}
+
+## Overview
+
+The Slot resource returns time slots from a schedule which are available for booking an appointment. Slots contain no information about actual appointments, only availability and type.
+
+When integrating your application with a client’s production environment you will work with the client to determine the Practitioner and Location ids (Millennium personnel and location codes, respectively) which they want to make available to third-party applications for enabling scheduling functionality.
+
+We understand this is a bit cumbersome, but we are always evaluating community feedback and look to improve the API in the future.
+
+The following fields are returned if valued:
+
+* [Slot id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Schedule](http://hl7.org/fhir/r4/slot-definitions.html#Slot.schedule){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/slot-definitions.html#Slot.status){:target="_blank"}
+* [Start](http://hl7.org/fhir/r4/slot-definitions.html#Slot.start){:target="_blank"}
+* [End](http://hl7.org/fhir/r4/slot-definitions.html#Slot.end){:target="_blank"}
+* [Scheduling Location Extension](#extensions)
+
+## Terminology Bindings
+
+<%= terminology_table(:slot, :r4) %>
+
+## Extensions
+
+* [Scheduling Location]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                    | Value\[x] Type | Description
+-----------------------|----------------|------------------------------------------------------------------------------------------------
+ `scheduling-location` | [`Reference`]  | Reference to the location corresponding to this schedule, where the appointment can be booked.
+
+## Search
+
+Search for Schedules that meet supplied query parameters:
```

Comment:
referred to Dstu2  documentation. we have the same thing there as well. Shall I change to Slots are is that fine? since we are including schedule as well.

Reply:
It can stay then if dstu2 is having the same thing